src/components/Button.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Button.css';
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  render() {
+    const { value } = this.props;
+    const { classStyle } = this.props;
+    return (
+      <button className={classStyle} type="submit">
+        {value}
+      </button>
+    );
+  }
+}
+
+Button.propTypes = {
+  value: PropTypes.string,
```

Comment:
This should be `required`. Without the `name` props, the button is useless, isn't it?
```suggestion
  value: PropTypes.string.required,
```